test/registries/PackageRegistry.test.ts

```diff
@@ -156,4 +156,11 @@ describe('PackageRegistry', () => {
 
     });
 
+    describe('#clear', () => {
+        it('should clear all packages', async () => {
+            const emptyDirStub: sinon.SinonStub = mySandBox.stub(fs, 'emptyDir').resolves();
```

Comment:
you don't need to stub this. If you add a package you can then check there are none when you call getAll